ios/FluentUI/People Picker/ContactCollectionView.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@objc(MSFContactCollectionView)
+open class ContactCollectionView: UICollectionView {
+    struct Constants {
+        static let sideInset: CGFloat = 16.0
+        static let extraSmallContentContactHeight: CGFloat = 115.0
+        static let smallContentContactHeight: CGFloat = 117.0
+        static let mediumContentContactHeight: CGFloat = 118.0
+        static let largeContentContactHeight: CGFloat = 121.0
+        static let extraLargeContentContactHeight: CGFloat = 125.0
+        static let extraExtraLargeContentContactHeight: CGFloat = 129.0
+        static let extraExtraExtraLargeContentContactHeight: CGFloat = 135.0
+    }
+
+    let layout: ContactCollectionViewLayout
+    @objc public weak var contactCollectionDelegate: UICollectionViewDelegate?
```

Comment:
I don't see this being used anywhere (maybe CMD+F doesn't work in github?)

Reply:
You're correct. I had forgotten to remove it since I initially thought that the delegate was mandatory when I was learning about `UICollectionView`s.